report_thesis/src/sections/background.tex

```diff
@@ -316,7 +316,59 @@ \subsubsection{Extra Trees Regressor (ETR)}
 As a tradeoff, \gls{etr} is less interpretable than a single decision tree, as the added randomness can introduce more bias than \gls{rf}.
 However, it often achieves better generalization performance, especially in high-dimensional or noisy datasets.
 
-\subsubsection{Gradient Boosting Regressor (GBR)}
+\subsubsection{Gradient Boosting Regression (GBR)}\label{sec:gradientboost}
+In this section we introduce \gls{gbr} primarily based on \citet{James2023AnIS}.
+\gls{gbr} is an ensemble learning technique that uses a sequence of weak learners, typically decision trees, to build a strong predictive model.
+
+However, in order to explain \gls{gbr}, it is helpful to build on the concepts of ensemble learning and decision trees.
+Ensemble learning is a technique in machine learning where multiple models, known as \textit{weak learners}, are combined to produce more accurate predictions.
+Mathematically, ensemble learning can be defined as combining the predictions of $M$ weak learners to form a final prediction $\hat{y}$, such that:
+\begin{equation}
+    \hat{y} = \sum_{m=1}^{M} \alpha_m \hat{y}_m,
+\end{equation}
+where $\hat{y}_m$ is the prediction of the $m$-th weak learner and $\alpha_m$ is the weight assigned to the $m$-th weak learner.
+While there are various choices for weak learners, decision trees are a common choice.
+Decision trees are a core component of gradient boosting methods. 
+They partition the data into subsets based on feature values, aiming to create groups where data points have similar predicted outcomes. 
+The quality of a split is measured using various metrics. Some common metrics include impurity, information gain, and Gini index. 
+
+Once optimal splits are identified, the tree is constructed by repeatedly partitioning the data until a stopping criterion is met.
+The final model consists of splits that create distinct regions, each with a predicted response value based on the mean of the observations in that region.
+
+This leads us to \textit{Gradient Boosting}.
+Instead of building a single decision tree, gradient boosting constructs multiple trees sequentially, with each new tree correcting the errors of the previous ones.
+Each tree is small, with few terminal nodes, preventing large adjustments based on a single tree's predictions. 
```

Comment:
```suggestion
Each tree is small, with few terminal nodes, preventing large adjustments based on a single tree's prediction.
```
Not sure but isn't this supposed to be "prediction" in singular?

Reply:
Chatgpts answer to this:
"The correct form is plural "predictions" since it refers to the multiple outcomes provided by a single tree."

But if you feel singular reads better, then I am all for it.